docs/static/releasenotes.asciidoc

```diff
@@ -39,6 +40,59 @@ This section summarizes the changes in the following releases:
 * <<logstash-8-0-0-alpha1,Logstash 8.0.0-alpha1>>
 
 
+[[logstash-8-9-0]]
+=== Logstash 8.9.0 Release Notes
+
+[[notable-8.9.0]]
+==== Notable issues fixed
+
+* Fix an issue where installs and updates of certain logstash plugins may fail behind a proxy https://github.com/elastic/logstash/pull/15131[#15131]. This issue was manifesting after a recent change to `logstash-filter-translate` updated to require the use of the `jar-dependencies` gem to retrieve artifacts from maven when installing the plugin, and could prevent the update of this plugin when a proxy is in use.
+* Improve logging when logstash is stalled on shutdown https://github.com/elastic/logstash/pull/15056[#15056]. This commit adds information about the main thread, if it is that one that is causing the shutdown to be stalled.
+* Improve SSL settings for connection to elasticsearch for central management and monitoring https://github.com/elastic/logstash/pull/15045[#15045]. This commit adds settings support for file-based certificates and cipher suites for management and monitoring settings, and removes the deprecation warnings from the logs that have been since SSL configuration settings were revamped in the Elasticsearch output.
```

Comment:
```suggestion
* Improved SSL settings for connection to {es} for central management and monitoring https://github.com/elastic/logstash/pull/15045[#15045]. 
This commit adds settings support for file-based certificates and cipher suites for management and monitoring settings, and removes the deprecation warnings from the logs that have been in since SSL configuration settings were revamped in the {es} output.
```